src/main/java/duke/command/AddEventCommand.java

```diff
@@ -0,0 +1,42 @@
+package duke.command;
+
+import duke.Parser;
+import duke.TaskList;
+import duke.UI;
+import duke.exception.LackOfDateException;
+import duke.exception.LackOfDescriptionException;
+
+public class AddEventCommand extends Command {
+    public AddEventCommand(String fullCommand) {
+        super(fullCommand);
+    }
+
+    private static void CheckDescription(String fullCommand) throws LackOfDescriptionException {
```

Comment:
Names representing methods should be written in camelCase.

Reply:
Thanks for reminds! Will make changes to the names of methods u pointed out in camelCase.

Reply:
Thanks for reminds! Will make changes to the names of methods u pointed out in camelCase.